variables.tf

```diff
@@ -1 +1,248 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# You must provide a value for each of these parameters.
+# ---------------------------------------------------------------------------------------------------------------------
 
+variable "name" {
+  type        = string
+  description = "Name of the vault stack, will be use to prefix resources"
+}
+
+variable "environment" {
+  type        = string
+  description = "Type of environment -- must be one of: dev, test, prod"
+}
+
+variable "key_pair_name" {
+  type        = string
+  description = "Keypair to associate to launched instances"
+}
+
+variable "ami_owner" {
+  type        = string
+  description = "Account id/alias of the AMI owner"
+}
+
+variable "additional_ips_allow_inbound" {
```

Comment:
this doesn't seem to be used?

Reply:
Going to remove it since there's no need for exposing it.